packages/app/cypress/e2e/files.cy.ts

```diff
@@ -0,0 +1,14 @@
+describe('App', () => {
```

Comment:
Did this app/cypress/integration get moved over?

Reply:
`app/cypress/integration/e2e` is now `app/cypress/e2e`. 

There was one other file that got removed: `packages/app/cypress/integration/visit-google.spec.ts`. It wasn't used for anything (filler from the early days where we needed a basic spec that would visit another domain).